src/components/datalist-item/__tests__/DatalistItem.test.tsx

```diff
@@ -1,66 +1,44 @@
 import React from 'react';
 import sinon from 'sinon';
-
+import { shallow } from 'enzyme';
 import DatalistItem from '..';
 
 const sandbox = sinon.sandbox.create();
-
 describe('components/datalist-item/DatalistItem', () => {
     afterEach(() => {
         sandbox.verifyAndRestore();
     });
-
     describe('render()', () => {
         test('should render default component', () => {
             const child = 'Test';
             const wrapper = shallow(<DatalistItem>{child}</DatalistItem>);
-
             expect(wrapper.is('li')).toBe(true);
             expect(wrapper.hasClass('datalist-item')).toBe(true);
-            expect(wrapper.prop('id')).toBeTruthy();
+            expect(wrapper.prop('id')).toMatch(new RegExp('^datalistitem?'));
```

Comment:
I don't know how often we use regular expressions but I think we'd prefer to use the literal version here.
```suggestion
            expect(wrapper.prop('id')).toMatch(/^datalistitem?/);
```